setup.cfg

```diff
@@ -24,8 +24,8 @@ install_requires =
     matplotlib>=3.5.2
     pandas>=1.4.2
     Pillow>=9.1.1
-    timm>=0.6.5
-    albumentations>=1.2.1
+    timm>=0.9.0
```

Comment:
It seems to be ok, because `timm` automatically includes `timm/` prefix for HF Hub.
Example: https://huggingface.co/timm/vit_small_patch14_dinov2.lvd142m